### ISSUE TITLE
Write tests/examples for collision routines

### ISSUE THREAD

**Issue by Fiona** [Fiona/Myrmidon#80]
I mean, we  can't really change them yet safely.

**Comment by Frimkron** [comment]
You'll find that _some_ of them have tests already

**Comment by Fiona** [comment]
True, and they're good too. But I really want a visual collision example to act as a sample.
